src/mongocrypt.h.in

```diff
@@ -869,7 +869,7 @@ mongocrypt_kms_ctx_bytes_needed (mongocrypt_kms_ctx_t *kms);
  * @param[in] bytes The bytes to feed. The viewed data is copied. It is valid to
  * destroy @p bytes with @ref mongocrypt_binary_destroy immediately after.
  * @returns A boolean indicating success. If false, an error status is set.
- * Retrieve it with @ref mongocrypt_ctx_status
+ * Retrieve it with @ref mongocrypt_kms_ctx_status
```

Comment:
Nice, while we're here, can you also remove the sentence "Retrieve it with @ref mongocrypt_ctx_status" from `mongocrypt_kms_ctx_status` below?

Reply:
Done!